spam/01_spam_tutorial.py

```diff
@@ -587,12 +644,15 @@ def has_person(x):
 
 
 # %% [markdown]
-# Because spaCy is such a common preprocessor for NLP (Natural Language Processing) applications, we also provide an alias for a `labeling_function` that uses spaCy. This resulting LF is identical to the one defined manually above.
+# Because spaCy is such a common preprocessor for NLP applications, we also provide a
+# [prebuilt `labeling_function` that uses spaCy](https://snorkel.readthedocs.io/en/master/source/snorkel.labeling.lf.html#snorkel.labeling.lf.nlp.nlp_labeling_function).
```

Comment:
prebuilt `labeling_function` decorator that uses spacy (or it sounds like it's a prebuilt LF).